doc/manuals/admin/build_source.md

```diff
@@ -42,6 +43,17 @@ commands that require root privilege):
         tar xfvz v1.0.2.tar.gz
         sudo mv rapidjson-1.0.2/include/rapidjson/ /usr/local/include
 
+* Install libwebsocket from sources:
+
+        wget https://github.com/warmcat/libwebsockets/archive/v1.6.3.tar.gz
+        tar xfvz v1.6.3.tar.gz
+        cd libwebsockets-1.6.3/
+        mkdir build
+        cd build
+        cmake ..          # default cmake sets /usr/local as installation path
+        make
+        sudo make install
+
```

Comment:
The procedure seems to be ok to compile in CentOS 6.x. However, it seems that (differently from Debian) /usr/local/lib is not included in the libary loading path so running compiled binary may result in

```
contextBroker: error while loading shared libraries: libwebsockets.so.6: cannot open shared object file: No such file or directory
```

The solution is described in http://serverfault.com/questions/372987/centos-usr-local-lib-system-wide-ld-library-path. However, it is not clear to me how to fit this in the RPM (including libwebsocket.so.6 as part of the CB RPM? In which directory? Create an ad hoc package RPM for the libwebsocket library and link packages through dependencies?)